### DISCUSSION TITLE
Authentication for additional hosts

### DISCUSSION THREAD

**Discussion by marioschaper** [homeassistant-apps/app-cloudflared#393] [Q&A] [answered]
Hi,

on my main cloudflare tunnel for thisisanexample.de i have an access policy configured. 
Now i've added an additional host in my HA cloudflare configuration that opens an internal http:// site. This site opens as subdomain.thisisanexample.de without the access policy working. 
What do i have to configure to enable a access policy for addiotional hosts?

kind regards
Mario

**Comment by brenner-tobias** [comment, marked as answer]
You have to create a separate, self-hosted Application in the Cloudflared Zero Trust Dashboard with that defined subdomain.
